sources/packages/backend/libs/utilities/src/queue.constant.ts

```diff
@@ -21,8 +21,7 @@ export enum QueueNames {
   FINProcessProvincialDailyDisbursementsIntegration = "fin-process-provincial-daily-disbursements-integration",
   PartTimeMSFAAProcessResponseIntegration = "part-time-msfaa-process-response-integration",
   FullTimeMSFAAProcessResponseIntegration = "full-time-msfaa-process-response-integration",
-  SFASSIntegration = "sfass-integration",
-  ATBCIntegration = "atbc-integration",
```

Comment:
this atbc request integration... when student click "request PD" btn.... why is it removed?

Reply:
The apply for PD status call to ATBC API is retained in API for the purpose that when a student is applying for PD from application, and once the ATBC API is called the Apply PD button needs to be disabled right away.

Hence removing the queue atbc-integration and it's queue configuration.